### PR TITLE
Add NH GRANIT orthoimagery and lidar hillshade layers

### PR DESCRIPTION
**Pull request by zekefarwell**
This adds orthoimagery and lidar hillshade imagery layers for the US state of New Hamshire available from [NH GRANIT](https://granit.unh.edu/), New Hampshire's statewide GIS clearinghouse.  I've confirmed with NH GRANIT that these datasets are available in the public domain and have no restrictions on their use ([documented on the OSM wiki here](https://wiki.openstreetmap.org/wiki/New_Hampshire#Usage_grant_for_NH_GRANIT_imagery)).  I've [added these same layers to the JOSM imagery sources](https://josm.openstreetmap.de/wiki/Maps/USA%20States?action=diff&version=335&old_version=331) as well.

### CODE REVIEW

**Comment by cicku** [comment]
Will get this reviewed in 2 weeks.

**Comment by cicku** [comment]
Please fix the error based on check result.

**Comment by zekefarwell** [comment]
OK I've added the available projections to fix the CI check.  I wouldn't have understood how to find those, but [a maintainer added them to the JOSM index](https://josm.openstreetmap.de/wiki/Maps/USA%20States?action=diff&version=336&old_version=335) so I copied them over.

Looks like now the CI check is throwing a different error.  Seems like maybe a multipolygon is not allowed for for the geometry?

**Comment by zekefarwell** [comment]
I figured out the geometry proplem and pushed a fix for that.  Unfortunately now there are several more errors.  It's saying the geojson ring orientation is incorrect and no connection adapters found for WMS servers.  I'm not sure what needs to be done to fix these errors.  The geojson files seem to load up just fine on [geojson.io ](https://geojson.io)

**Comment by zekefarwell** [comment]
Ok looks like I've managed to resolve all the geometry problems.  Now there's just the "no connection adapters found for WMS servers" error.  I don't know what this means but am happy to fix it if you can point me in the right direction.

File: sources/north-america/us/nh/LiDAR_Bare_Earth_NE_HS_NH_2022.geojson [hunk 1]
```diff
@@ -0,0 +1,989 @@
...
+    "properties": {
+        "id": "LiDAR_Bare_Earth_NE_HS_NH_2022",
+        "attribution": {
+            "url": "https://granit.unh.edu/",
+            "text": "NH GRANIT",
+            "required": false
+        },
+        "name": "NH GRANIT LiDAR Hillshade 2022 (Northeast)",
+        "icon": "https://www.arcgis.com/sharing/rest/content/items/9077b6cdf1d94589b79cc66d8f919da2/resources/GRANITLogoTransparent1.png",
+        "url": "CDATA[https://nhgeodata.unh.edu/nhgeodata/services/ImageServices/LiDAR_Bare_Earth_NE_HS_NH_2022/ImageServer/WMSServer?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=0&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
```
Comment (arch0345) [inline sources/north-america/us/nh/LiDAR_Bare_Earth_NE_HS_NH_2022.geojson]:
```suggestion
        "url": "https://nhgeodata.unh.edu/nhgeodata/services/ImageServices/LiDAR_Bare_Earth_NE_HS_NH_2022/ImageServer/WMSServer?FORMAT=image/png&TRANSPARENT=TRUE&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap&LAYERS=0&STYLES=&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}",
```

**Review by arch0345 — COMMENTED** [review]
This seems to be where the error is coming from

File: sources/north-america/us/nh/LiDAR_Bare_Earth_NE_HS_NH_2022.geojson (hunk 1 above)
Comment (zekefarwell) [reply sources/north-america/us/nh/LiDAR_Bare_Earth_NE_HS_NH_2022.geojson]: Ahh.  Of course.  Bad coversion from the original JOSM xml.  Thank you!

**Comment by zekefarwell** [comment]
It looks like all the checks are passing now.  Do you see any other issues that need to be resolved, @cicku?

**Pull request merged**